index.js

```diff
@@ -149,6 +149,52 @@ class GithubScm extends Scm {
     }
 
     /**
+     * Checkout the source code from a repository; resolves as an object with checkout commands
+     * @method getCheckoutCommand
+     * @param  {Object}    config
+     * @param  {String}    config.branch        Pipeline branch
+     * @param  {String}    config.host          Scm host to checkout source code from
+     * @param  {String}    config.org           Scm org name
+     * @param  {String}    config.repo          Scm repo name
+     * @param  {String}    config.sha           Commit sha
+     * @param  {String}    [config.prRef]       PR reference (can be a PR branch or reference)
+     * @return {Promise}
+     */
+    _getCheckoutCommand(config) {
+        const checkoutUrl = `https://${config.host}/${config.org}/${config.repo}`;
+        const checkoutSha = config.prRef ? config.branch : config.sha; // if PR, use pipeline branch
```

Comment:
Should rename this to be `checkoutRef` to be less confusing, since it could be a branch or a sha.